Step.txt

```diff
@@ -56,12 +56,14 @@ Process of Creating the script
 2. In the next Step, we are going to print out report in this following
     If we select one of the test case, and in of the battery type is selected, it is going to print out stream lenght (50 now) output one by one in console
     eg : Sender Battery Type, Battery Index, Temperature Type, Temperature Detected
```

Comment:
format of multiple parameters is not defined in test cases also in steps.txt

Reply:
Hi @sunilgowda95
Thanks for the comment, although the multiple parameter is not part of the basic sender requirement, only streaming 50 (or any user defined input) data would be sufficient,
do let me know what to be done to modify the solution

Reply:
Please check the latest changes

Reply:
Thank you. Now I can see from Actions output.